.eslintrc

```diff
@@ -0,0 +1,11 @@
+{
+    "extends" : "airbnb-typescript/base",
```

Comment:
Is this consistent across other projects?  It seems risky to arbitrarily point upstream to another source that can change at any time.

Reply:
I guess it is picked up via npm upgrade so it would go through PR build checks.

Reply:
Yeah I have been thinking about this a little hence why its just a draft at the moment. A lot of people tend to follow the airbnb style guide, however there are other options such as @typescript-eslint/recommended. I certainly think we should be extending a base set of configs rather than writing our own style guide since writing our own would be a lot of work for very little benefit

Reply:
Will give `plugin:@typescript-eslint/recommended` a go a little later since airbnb is flagging a fair few issues that need overriding in this project

Reply:
Should prettier be done first?  Disable tslint, prettier, then enable eslint?

Reply:
It's been a while but I think for tslint I took the ruleset from TypeScript and then disabled the few that were "being difficult".

Reply:
> It's been a while but I think for tslint I took the ruleset from TypeScript and then disabled the few that were "being difficult".

Okay I will try the typescript recommended settings since I think they are most similar

Reply:
> Should prettier be done first?  Disable tslint, prettier, then enable eslint?

Don't think it makes too much difference either way. This way they can be reviewed separately without leaving the code without a linter for a period of time